pulser/simulation/simulation.py

```diff
@@ -635,44 +636,64 @@ def _construct_hamiltonian(self, update_and_extract: bool = True) -> None:
         if not hasattr(self, "basis_name"):
             self._build_basis_and_op_matrices()
 
-        def make_vdw_term() -> qutip.Qobj:
+        def make_vdw_term(
+            q1: Union[int, str], q2: Union[int, str]
```

Comment:
You can use the `QubitId` you imported from `pulser.register` for the type hints here